ciphers/a1z26.py

```diff
@@ -7,15 +7,18 @@
 """
 
 
-def encode(plain: str) -> list[int]:
```

Comment:
`from __future__ import annotations`

and then undo all the rest of these changes.

Reply:
I meant to open a new pull request to fix mypy issues. I will try to undo all changes. Sorry for that.